src/rgw/rgw_admin.cc

```diff
@@ -4247,10 +4247,26 @@ int main(int argc, const char **argv)
     bool need_cache = readonly_ops_list.find(opt_cmd) == readonly_ops_list.end();
     bool need_gc = (gc_ops_list.find(opt_cmd) != gc_ops_list.end()) && !bypass_gc;
 
+    std::string rgw_store = "rados";
+    const auto& config_store = g_conf().get_val<std::string>("rgw_backend_store");
+    #ifdef WITH_RADOSGW_DBSTORE
+    if (config_store == "dbstore") {
```

Comment:
either we should use if elseif or switch statements. Is there any specific need to check for db_store and then for motr both? Probably not that important as of today, but assume in future we need to support 10 different stores, we would definitely not like 10 if statements as well as if else statements